scripts/documentation_provider.py

```diff
@@ -50,6 +50,46 @@ def __init__(self) -> None:
         self.printed_entries: List[str] = []
         with open("api.json") as json_file:
             self.api = json.load(json_file)
+        self.api["StreamIO"] = {
```

Comment:
I don't think we need these - these classes are not a part of the API, they don't need to be exposed. Just a helper class that converts channel into the stream is seemingly all we need.